cost/google/schedule_instance/google_schedule_instance.pt

```diff
@@ -367,8 +491,14 @@ script "js_instances_with_schedule", type: "javascript" do
       })
     }
 
-    if (schedule != null) {
-      // schedule="0900-0930_mo-tu-we-th-fr_america-los_angeles"
+    // Check that we have a schedule and it is not explicitely disabled before attempting to parse the schedule tag value
+    // Test the regex pattern for a valid schedule as well, in case the tag value is not a valid schedule
+    // schedule="0900-0930_mo-tu-we-th-fr"
+    // schedule="0900-0930_mo-tu-we-th-fr_america-los_angeles"
+    var schedule_is_valid = /([0-9]{1,2}.*)-([0-9]{1,2}.*)_([a-z\-]*).*/.test(schedule)
+    if ((_.isString(schedule)) && (schedule_is_valid == false)) {
+      console.log("Skipping instance " + instance['instanceId'] + " with invalid schedule '" + schedule +"'");
```

Comment:
Same as `console.log()` comments on AWS policy above

Reply:
Using this thread to discuss:
https://github.com/flexera-public/policy_templates/pull/2817#discussion_r1834542127